### ISSUE TITLE
SimCSE: Simple Contrastive Learning of Sentence Embeddings

### ISSUE THREAD

**Issue by mingqook** [mingqook/Reading_Paper#36]
## Abstract (초록) 📝
This paper presents SimCSE, a simple contrastive learning framework that greatly advances state-of-the-art sentence embeddings. We first describe an unsupervised approach, which takes an input sentence and predicts itself in a contrastive objective, with only standard dropout used as noise. This simple method works surprisingly well, performing on par with previous supervised counterparts. We find that dropout acts as minimal data augmentation, and removing it leads to a representation collapse. Then, we propose a supervised approach, which incorporates annotated pairs from natural language inference datasets into our contrastive learning framework by using "entailment" pairs as positives and "contradiction" pairs as hard negatives. We evaluate SimCSE on standard semantic textual similarity (STS) tasks, and our unsupervised and supervised models using BERT base achieve an average of 76.3% and 81.6% Spearman's correlation respectively, a 4.2% and 2.2% improvement compared to the previous best results. We also show -- both theoretically and empirically -- that the contrastive learning objective regularizes pre-trained embeddings' anisotropic space to be more uniform, and it better aligns positive pairs when supervised signals are available.

## 논문(Paper)에서 다루는 핵심 개념 (Idea) 💡
* SimCSE, Sentence embedding, Contrastive learning

## 논문(Paper) 정리 
* https://velog.io/@mingqook/SimCSE-Simple-Contrastive-Learning-of-Sentence-Embeddings

## 논문(Paper) 원본
* https://arxiv.org/abs/2104.08821
